src/main/java/http/HttpClient.java

```diff
@@ -129,78 +116,25 @@ public String post(String sufixPath, String jsonBody) {
 
     }
 
-    public Boolean testApiKey(){
-        String response = this.get("v2/running_analyses_by_api_key");
-        IResponseInfo responseInfo = this.helpers.analyzeResponse(response.getBytes());
-        if( responseInfo.getStatusCode() == 200){
-            this.util.sendStdout("API Key OK!");
-            return true;
-        }else{
-            this.util.sendStderr("API Key NOT OK!");
-            this.util.sendStderr("Status code: "+responseInfo.getStatusCode());
-            this.util.sendStderr(response.substring(responseInfo.getBodyOffset()));
-            return false;
-        }
-    }
-
-    public IResponseInfo postMultiForm(String suffixPath, HttpEntity httpMultipartEntity){
 
+    public HttpResponse postMultiForm(HttpEntity httpMultipartEntity){
         try {
 
             CloseableHttpClient httpClient = HttpClients.createDefault();
-            HttpPost httpPost = new HttpPost("https://"+CONVISO_API_HOST+CONVISO_API_PATH+suffixPath);
+            HttpPost httpPost = new HttpPost("https://"+CONVISO_API_HOST+CONVISO_API_PATH);
 
-            /*HttpHost proxy = new HttpHost("127.0.0.1", 8080, "http");
-            RequestConfig config = RequestConfig.custom().setProxy(proxy).build();
-            httpPost.setConfig(config);*/
+//            HttpHost proxy = new HttpHost("127.0.0.1", 8080, "http");
```

Comment:
Shouldn't we just remove commented lines of code?

Reply:
This one i would like to maintain because of a feature in the future. But will also be removed soon.